src/framework/audio/driver/platform/win/wasapiaudiodriver.cpp

```diff
@@ -336,7 +337,9 @@ void WasapiAudioDriver::updateAudioDeviceList()
     }
 
     auto devices = audioDevices(m_data->enumerator);
-    m_deviceList = devices.first;
+    m_deviceList.clear();
+    m_deviceList.push_back({ DEFAULT_DEVICE_ID, muse::trc("audio", "System default") });
+    m_deviceList.insert(m_deviceList.begin(), devices.first.begin(), devices.first.end());
```

Comment:
Shouldn't it rather be m_deviceList.insert(m_deviceList.**end()**, ...); ?

Reply:
oops